lucas-varela/Bodiless-JS
# feat(cx-structuredata): Provide Structure Data Package

## Overview
Provides new package, structure data, to make it possible to create JSON+LD from components data. Note this is not activated on starter site yet. Needs to add the provider to the pages, and apply components tokens so that the module can extract data.

## Changes
- Created `cx-structuredata` package.
- Added the package to `tsconfig.settings.json` at root.
- The package contains the following schemas: FAQ, Web & Organization, Product, and Video.

## Test Instructions
- Only YouTube video schema is in scope. To make it work, needs to set locally google video api key variable (`BODILESS_GOOGLE_YOUTUBE_API_KEY`) so that the service can retrieve proper video data (See JIRA ticket).

## Related Issues
- #1430
